test/test_helper.rb

```diff
@@ -1,8 +1,9 @@
 ENV["RAILS_ENV"] ||= "test"
 require File.expand_path('../../config/environment', __FILE__)
-require "rails/test_help"
-require "capybara/rails"
-require "minitest/emoji"
+require 'rails/test_help'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.